projects/coordinator/src/test/java/org/batfish/coordinator/WorkMgrServiceTest.java

```diff
@@ -89,13 +91,33 @@ public void getNonEmptyContainer() throws Exception {
   }
 
   @Test
-  public void testDeleteQuestionsFromAnalysis() throws Exception {
+  public void testConfigureAnalysis() throws Exception {
```

Comment:
These two tests are actually very long, but since we need to initialize analysis and add questions before testing deleting them, I make these tests together. I think we will split this function into three parts later (initAnalysis, addAnalysisQuestion, delAnalysisQuestion).